e2e/makefile

```diff
@@ -0,0 +1,54 @@
+# Variables
+DOCKER_COMPOSE_FILE = helpers/docker/docker-compose.yml
+NESTJS_SERVICE_PORT = 3000
+NESTJS_TEST_CMD = npm run test:e2e 
+MAX_RETRIES = 10 # Number of retries to check if the inddexer service is up
+RETRY_INTERVAL = 5 # Seconds between retries
+PM2_CMD = pm2
+NPM_CMD = npm
+APP_NAME = silent-indexer
+
+# Targets
+.PHONY: all up down test wait-for-service clean start-service
+
+# Default target
+all: test clean
+
+# Start Docker Compose services
+up: down
+	@echo "Starting Docker Compose services..."
+	docker compose -f $(DOCKER_COMPOSE_FILE) up -d
+
+# Stop and remove Docker Compose services
+down:
+	@echo "Stopping and removing Docker Compose services..."
+	docker compose -f $(DOCKER_COMPOSE_FILE) down
```

Comment:
```suggestion
	docker compose -f $(DOCKER_COMPOSE_FILE) down -v
```

Reply:
The `-v` option is meant to remove named volumes, which we do not use.

Reply:
It also removes anonymous volumes which are used in the image.